src/aircraft_cmd.cpp

```diff
@@ -1556,6 +1557,7 @@ static void AircraftEventHandler_AtTerminal(Aircraft *v, const AirportFTAClass *
 			if (v->subtype == AIR_HELICOPTER && apc->num_helipads > 0) {
 				/* an excerpt of ServiceAircraft, without the invisibility stuff */
 				v->date_of_last_service = _date;
```

Comment:
I'll make a separate PR later, following the approach of #10721. I guess I'll also need to do the same for `v->build_year` since NewGRF can get that too. 😕 (Writing it here so I don't forget! 😛)